### PR TITLE
Solr 8.7

### PR DESCRIPTION
**Pull request by dsmiley**
https://cwiki.apache.org/confluence/display/SOLR/ReleaseNote87
I followed the "update.md" script, nearly without issue.  My docker image space filled up which caused confusing errors, but once I fixed that, everything built and then tested okay.

### CODE REVIEW

File: TAGS [hunk 1]
```diff
@@ -18,3 +18,5 @@
 6.6/slim:6.6.6-slim:6.6-slim 6-slim
 5.5:5.5.5:5.5 5
 5.5/slim:5.5.5-slim:5.5-slim 5-slim
+8.7:8.7.0:8.7 8 latest
+8.7/slim:8.7.0-slim:8.7-slim 8-slim latest-slim
```
Comment (makuk66) [inline TAGS:22]: These should be sorted at the top; the file appears to be in descending version order. Did the scripts not do that?

**Review by makuk66 — CHANGES_REQUESTED** [review]

File: TAGS (hunk 1 above)
Comment (dsmiley) [reply TAGS:22]: I simply ran the scripts, following update.md instructions.

**Review by makuk66 — APPROVED** [review]

File: TAGS (hunk 1 above)
Comment (dsmiley) [reply TAGS]: I moved it manually.  Can you approve please?

**Comment by dsmiley** [comment]
Here's the PR to official-images: https://github.com/docker-library/official-images/pull/9045

**Pull request merged**